sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -313,9 +313,7 @@ def copy(self, src, dest):
       self.client.objects.Copy(request)
     except HttpError as http_error:
       if http_error.status_code == 404:
-        # This is a permanent error that should not be retried. Note that
-        # FileBasedSink.finalize_write expects an IOError when the source
-        # file does not exist.
+        # This is a permanent error that should not be retried.
         raise GcsIOError(errno.ENOENT, 'Source file not found: %s' % src)
```

Comment:
<!--new_thread; commit:24c81c1018b47c5316f09f3b1eef8725b5492848; resolved:0-->
Do we still need to re-wrap this error ?